README.md

```diff
@@ -1,16 +1,42 @@
 # JDBC Connector
 
-[![Build Status](https://travis-ci.com/modakanalytics/jdbc.almaren.svg?token=TEB3zRDqVUuChez9334q&branch=master)](https://travis-ci.com/modakanalytics/jdbc.almaren)
+[![Build Status](https://github.com/modakanalytics/jdbc.almaren/actions/workflows/Jdbc.almaren-githubactions.yml/badge.svg)](https://github.com/modakanalytics/jdbc.almaren/actions/workflows/Jdbc.almaren-githubactions.yml)
 
 JDBC Connector allow you to execute any SQL statement using Apache Spark.
 
+To add JDBC connector dependency to your sbt build:
+
 ```
-libraryDependencies += "com.github.music-of-the-ainur" %% "jdbc-almaren" % "0.0.4-3.1"
+libraryDependencies += "com.github.music-of-the-ainur" %% "jdbc-almaren" % "0.0.5-3.3"
 ```
+To run in spark-shell:
 
+For scala-version(2.12.15):
+```
+spark-shell --master "local[*]" --packages "com.github.music-of-the-ainur:almaren-framework_2.12:0.9.8-3.3,com.github.music-of-the-ainur:jdbc-almaren_2.12:0.0.5-3.3"
 ```
-spark-shell --master "local[*]" --packages "com.github.music-of-the-ainur:almaren-framework_2.12:0.9.3-$SPARK_VERSION,com.github.music-of-the-ainur:jdbc-almaren_2.12:0.0.4-$SPARK_VERSION"
+For scala-version(2.13.9):
```

Comment:
change to scala 2.13 don't keep the minor version do the same changes above.

Please do the same changes for spark-3.3